api/base/serializers.py

```diff
@@ -44,6 +48,16 @@ def format_relationship_links(related_link=None, self_link=None, rel_meta=None,
     return ret
 
 
+def is_anonymized(request):
+    is_anonymous = False
+    private_key = request.query_params.get('view_only', None)
+    if private_key is not None:
+        link = PrivateLink.find_one(Q('key', 'eq', private_key))
```

Comment:
PrivateLink.find_one throws a NoResultsFound exception so `/v2/nodes/3wkv5/?view_only=woot_woot&format=jsonapi` crashes to a django stack trace, assuming of course that you do not have a PrivateLink with a key of `woot_woot` in your database.